vscode/microsoft-kiota/src/kiotaInterop.ts

```diff
@@ -0,0 +1,129 @@
+import * as vscode from "vscode";
+import * as cp from 'child_process';
+import * as rpc from 'vscode-jsonrpc/node';
+
+export async function connectToKiota<T>(callback:(connection: rpc.MessageConnection) => Promise<T | undefined>): Promise<T | undefined> {
+  const childProcess = cp.spawn("C:\\sources\\github\\kiota\\src\\kiota\\bin\\Debug\\net7.0\\kiota.exe", ["rpc"],{
```

Comment:
Is this hardcoded path going to impact portability?

Reply:
yes it will, I still need to figure out how to embed kiota into the extension, which will be part of the CI/CD efforts

Reply:
Added a comment to https://github.com/microsoft/kiota/issues/2333 in the decription to make it clear the CI/CD will look into it.